theme/static/js/hs_file_browser.js

```diff
@@ -1137,7 +1137,7 @@ function setupModelInstanceTypeUI() {
             // optional fields are not displayed by default in JSONEditor form for editing
             // user needs to select any optional properties to make it available for editing
             display_required_only: false,
-            required_by_default: true,
+            required_by_default: false,
```

Comment:
This is further reaching than just MODFLOW.  What is the reason for this change to the jsoneditor?

Reply:
This is the default setting for the jsoneditor and it should have used been used from the beginning. Having `required_by_default: true` makes all fields as required if the `required` property is missing for those fields in the schema. In the case of a schema where all fields are optional, this will make all fields required in the editor. By having `required_by_default: false`, only the fields in the schema that have required property, will be set by the editor as required. 

I think I need to change the settings for `display_required_only` to `true` so that only required fields are displayed in the editor by default.